jax/experimental/jax2tf/tests/shape_poly_test.py

```diff
@@ -2567,6 +2580,10 @@ def test_harness(self, harness: PolyHarness):
         raise unittest.SkipTest(
             "native lowering with shape polymorphism not implemented for JAX primitives still using HLO fallback lowering; b/261682623")
 
+      if harness.fullname == "jnp.cumsum_reduce_axis=poly" and jtu.device_under_test() == "tpu":
+        # https://github.com/openxla/stablehlo/issues/1258
+        raise unittest.SkipTest(
+            "native lowering with shape polymorphism not implemented for window_reductions on TPU")
```

Comment:
same question as before.  Since shape_poly_test.py can run both non_native_lowering and native_lowering.   so Non-native-lowering with TPU will also skip this unit test. 
Do we need mention "native_lowering" in the error message?

Reply:
This whole conditional is under a bigger conditional that only applies on native lowering.